site/app/models/Config.php

```diff
@@ -38,6 +38,8 @@
  * @method \DateTimeZone getTimezone()
  * @method string getUploadMessage()
  * @method array getHiddenDetails()
+ * @method void loadMasterIni()
+ * @method void loadCourseIni()
```

Comment:
you only need to add functions that are done via the magic `__call` function in AbstractModel as I use these for intellisense in IntelliJ as well as for constructing mocks of these models in the test framework.